doc/manual/src/language/constructs.md

```diff
@@ -1,5 +1,372 @@
 # Language Constructs
 
+## Basic Literals
+
+### String {#string-literal}
```

Comment:
I like headers over `a`; they have a better UX when linking. :+1: 

I was going to suggest to preserve the anchor id, but `type-string` would be really bad.
Could you add redirects though?

Reply:
`#type-string` is alive and well and living in `values.md`, and any of this content relevant to strings as a data type may migrate back over there. It's a bit unfortunate that web links pointing to `values.md#type-string` will point to different content now, but there's a link to `constructs.md#string-literal` over there so readers don't get lost.